### DISCUSSION TITLE
Ansible builds

### DISCUSSION THREAD

**Discussion by ch4r13** [aluminum-ice/pwnagotchi#51] [General]
Hello,
I would like to ask you, how you are build the pwnagotchi to final state ? I understand how Ansible works, just basically. So you build it  with playbook over SSH connected to Raspberry zero2w physically ? After that, put the microSD card out and pack it in 7z, or there is another way to emulate it on your x64 computer ? 
Thanks for reply, I'm just trying to understand the scenario.

**Comment by aluminum-ice** [comment]
No, its built on a regular x86 machine running ubuntu. Read the Makefile that shows exactly how it's built
